### PR TITLE
feat: Add Elixir docs

### PR DESCRIPTION
**Pull request by nand4011**
Add an Elixir SDK section in the docs with a cheat sheet.

Wire in Elixir snippet parsing.

### CODE REVIEW

**Comment by nand4011** [comment]
The build will pass as soon as https://github.com/momentohq/client-sdk-elixir/pull/43 is merged

**Comment by NoSQLKnowHow** [comment]
I added an image to the index page. I also fixed some things that were uppercase, but shouldn't have been to keep with our doc's style.

File: docs/develop/sdks/elixir/index.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Welcome to the Momento Elixir SDK documentation!
+<img src="/img/elixir.jpg" width="90%" alt="a picture of a fantasy dwarf with an water-based elixir." />
+
+The Momento Elixir SDK is available via the hex package [`gomomento`](https://hex.pm/packages/gomomento).
+
+The source code can be found on GitHub: [momentohq/client-sdk-elixir](https://github.com/momentohq/client-sdk-elixir).
+
+## Requirements
+
+- Elixir version 1.14 or higher is required.
```
Comment (NoSQLKnowHow) [inline docs/develop/sdks/elixir/index.md]:
```suggestion
- [Elixir version 1.14](https://elixir-lang.org/install.html) or higher is required.
```

Is this the correct URL to add here?

**Review by cprice404 — DISMISSED** [review]
LGTM as soon as @NoSQLKnowHow is happy!

File: docs/develop/sdks/elixir/index.md (hunk 1 above)
Comment (nand4011) [reply docs/develop/sdks/elixir/index.md]: Indeed it is.

**Review by cprice404 — APPROVED** [review]

**Pull request merged**